packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -233,7 +234,10 @@ class SingleChildScrollView extends StatelessWidget {
   final String? restorationId;
 
   /// {@macro flutter.widgets.scroll_view.keyboardDismissBehavior}
-  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  ///
+  /// If [keyboardDismissBehavior] is null then it will fallback to the closest
+  /// [ScrollConfiguration] provided by the context.
```

Comment:
```suggestion
  /// If [keyboardDismissBehavior] is null then it will fallback to the inherited
  /// [ScrollBehavior.keyboardDismissBehavior].
```